frontend/src/pages/Order/Order.tsx

```diff
@@ -0,0 +1,32 @@
+import { FC } from "react";
+import { OrderProps } from "../Account/components/OrderPreview/types";
+
+const formatDate = (date: Date) => {
+  const dayOfWeek = date.toLocaleDateString("en-US", { weekday: "long" });
+  const day = date.toLocaleDateString("en-US", { day: "2-digit" });
+  const month = date.toLocaleDateString("en-US", { month: "short" });
+  return `${dayOfWeek} ${day} ${month}`;
```

Comment:
These functions are identical. It would be better to extract them into a separate file to avoid duplication.